lib/watir/elements/iframe.rb

```diff
@@ -125,7 +125,6 @@ def send_keys(*args)
     def switch!
       @driver.switch_to.frame @element
       @browser.default_context = false
-      @browser.after_hooks.run
```

Comment:
Why do we need to drop this?

Reply:
executing the hooks when changing the context to the frame was causing a bunch of issues